Update Harvester v1.0 install configs (#98)
harvester/docs: docs/rancher/virtualization-management.md

For Harvester v0.3.0 and above, virtualization management with the multi-cluster management feature will be supported using Rancher v2.6.x.

> 1. Firstly, you will need to spin up a Rancher with v2.6.3 or above of Harvester v1.0.0 management. For testing purposes, you can spin up a Rancher server using the following `docker run` command: 

Reviewer: ```suggestion
1. As a prerequisite, you will need to create a Rancher server with v2.6.3 or above with Harvester v1.0.0 management. For testing purposes, you can spin up a Rancher server using the following `docker run` command: 
```

1. Create a VM with one of the images that you have uploaded.
1. Log in with another user, e.g., `project-readonly`, and this user will only have the read permission of this project.

## Delete Imported Harvester Cluster

!!! note

>     You will need to reset the `cluster-registration-url` to empty on the setting page of the imported Harvester cluster first. A known issue will be addressed via [#1686](https://github.com/harvester/harvester/issues/1686).

Reviewer 2: This doesn't seem to be necessary. Removing the cluster from rancher dashboard should be enough

Author: IIRC, the Rancher will not delete the agent node pod on the downstream Harvester cluster, and rebooting the agent node will add the cluster back unexpectedly.

Reviewer 2: No, it won't add the cluster back on restart. The cluster agent is in charge of the connection, it does not create clusters.

Author: cool, thanks for the clarification.

>     You will need to reset the `cluster-registration-url` to empty on the setting page of the imported Harvester cluster first. A known issue will be addressed via [#1686](https://github.com/harvester/harvester/issues/1686).

Reviewer: ```suggestion
    You need to delete, manually, the `cluster-registration-url` value in the settings page of the imported Harvester cluster first. A known issue is open for addressing this bug: [#1686](https://github.com/harvester/harvester/issues/1686).
```
